assets/js/app.js

```diff
@@ -29,7 +29,7 @@ let csrfToken = document.querySelector("meta[name='csrf-token']").getAttribute("
 let liveSocket = new LiveSocket("/live", Socket, {params: {_csrf_token: csrfToken}})
 
 // Show progress bar on live navigation and form submits
-topbar.config({barColors: {0: "#29d"}, shadowColor: "rgba(0, 0, 0, .3)"})
+topbar.config({barColors: {0: "#f59e0b"}, shadowColor: "rgba(0, 0, 0, .3)"})
```

Comment:
To match the default config color of the project.

_It can be changed in the future._